pkg/antctl/raw/supportbundle/command.go

```diff
@@ -523,8 +580,41 @@ func controllerRemoteRunE(cmd *cobra.Command, args []string) error {
 		defer f.Close()
 		io.Copy(f, reader)
 	}
-	if err := requestAll(agentClients, controllerClient, bar); err != nil {
+	results := requestAll(agentClients, controllerClient, bar)
+	resultMap := downloadAll(agentClients, controllerClient, dir, bar, results)
+
+	return processResults(resultMap, dir)
+}
+
+// processResults will output the failed nodes and their reasons if any. If no data was collected,
+// error is returned, otherwise will return nil.
+func processResults(resultMap map[string]error, dir string) error {
+	resultStr := ""
+	var failedNodes []string
+	allFailed := true
+	var err error
+
+	for k, v := range resultMap {
+		if k != "*" && v != nil {
+			resultStr += fmt.Sprintf("- %s: %s\n", k, v.Error())
+			failedNodes = append(failedNodes, k)
```

Comment:
Do you think we can also output the error if we fail to collect controller logs?

Reply:
Updated.